AdyenSession/API/Partial Payments API/Create Order/CreateOrderRequest.swift

```diff
@@ -0,0 +1,63 @@
+//
+// Copyright (c) 2022 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Adyen
+import AdyenNetworking
+import Foundation
+
+internal struct CreateOrderRequest: APIRequest {
+
+    internal typealias ResponseType = CreateOrderResponse
+
+    internal let path: String
+
+    internal var counter: UInt = 0
+
+    internal let method: HTTPMethod = .post
+
+    internal let headers: [String: String] = [:]
+
+    internal let queryParameters: [URLQueryItem] = []
+
+    internal let sessionData: String
+    
+    internal init(sessionId: String,
+                  sessionData: String) {
+        self.path = "checkoutshopper/v1/sessions/\(sessionId)/orders"
```

Comment:
Shouldn't we have this `checkoutshopper/v1/sessions/` part as a constant. Would be annoying to change each one when there is a version upgrade

Reply:
can I fix this in an another PR?, its used in all other checkoutshopper requests